_data/companies.yml

```
@@ -286,11 +286,11 @@
   last_update: 2020-03-05
 
 - name: Culture Amp
-  wfh: Encouraged
+  wfh: Restricted
```

```suggestion
  wfh: Required
```

Should that be required rather than restricted?